app/javascript/modules/listingDetailsLottery/LotteryDetailsLotteryInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { SidebarBlock, t } from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { localizedFormat, renderWithInnerHTML } from "../../util/languageUtil"
+import { isLotteryComplete, isOpen } from "../../util/listingUtil"
+
+export interface ListingDetailsLotteryInfoProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsLotteryInfo = ({ listing }: ListingDetailsLotteryInfoProps) => {
+  if (isOpen(listing) || !listing.Lottery_Date || isLotteryComplete(listing)) {
+    return null
+  }
+
+  return (
+    <>
+      <div className="border-b border-gray-400 md:border-b-0">
+        <SidebarBlock title={t("label.lottery")}>
+          <p className="flex justify-between mb-4">
+            <span>{localizedFormat(listing.Lottery_Date, "LL")}</span>
+            <span className="font-bold">{localizedFormat(listing.Lottery_Date, "LT")}</span>
+          </p>
+          {listing.Lottery_Venue && renderWithInnerHTML(listing.Lottery_Venue)}
```

Comment:
noticed the text color here differs slightly from the block below/angular — I think we want to wrap this with an element with `class="text-gray-750"` to match